groepjes.sh

```diff
@@ -38,7 +38,8 @@ for dir in "$@"; do
 
 	#TOID=`sed -n '/Name:/s/.*\([sez][0-9]\+\).*/\1/p' "$file"`
 	#TOID=`grep -oi '\<[sez]\?[0-9]\{6,7\}\>' "$file" | tr SEZ sez | sort -u`
-	TOID=`find "${file%%/*}" -type f -not -name "*.WARNING" | xargs grep -oihI '\<[usefz]\?[0-9]\{6,7\}\>' | sed 's/\<[0-9]/s&/' | tr USEFZ usefz`
+  #TOID=`grep -oihI '\<[usefz]\?[0-9]\{6,7\}\>' "${file%%/*}"/* | sed 's/\<[0-9]/s&/' | tr USEFZ usefz`
+	TOID=`find "${file%%/*}" -type f -not -name "*.WARNING" | xargs grep -oihI '\<[usefz]\?[0-9]\{6,7\}\>' | sed 's/\<[0-9]/s&/' | tr USEFZ usefz
```

Comment:
I tried using the Github-editor to resolve the conflict. My mistake. Fixed in 32df16659fd0a9bfe31c5b52a6ca2185471244cf